### DISCUSSION TITLE
[Cancelled] : Core Working Group Meeting on January 14th, 2022

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#188] [Announcements]
## Agenda

- Formal consensus on opening up protocol license from CC-BY-ND to CC-BY-SA from version 0.9.4

## Attendees

- @pramodkvarma 
- @core-wg-admin 
- @pkapustin 
- @venkatramanm 
- @sjthnrk 

## Meeting Time
5:00 pm IST

## Duration
30 minutes

**Comment by pramodkvarma** [comment]
Could we also update currently used versions (not just future versions) to enable the community to have less restrictive license? Thanks.

**Reply by ravi-prakash-v** [reply]
Yeah I think we can. The licensing policy of Github states that the latest license on the repo will override all previous licenses. So we should be good.

**Comment by ravi-prakash-v** [comment]
Cancelling this call as it is a holiday on Friday, 14th January.
